src/components/PersonElement.tsx

```diff
@@ -0,0 +1,84 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+import { Person } from '../types/Person';
+
+interface Props {
+  person: Person,
+  handleSelection: (name: string | undefined, slug: string,) => void,
+  selectedName: string | undefined,
+}
+
+export const PersonElement: React.FC<Props> = ({
+  person,
+  handleSelection,
+  selectedName,
+}) => {
+  return (
+    <tr
+      key={person.name}
```

Comment:
Remove the key attribute from the tr element, as it is not needed in this context.